### ISSUE TITLE
Language of Parts in Video Captions

### ISSUE THREAD

**Issue by brennanyoung** [w3c/wcag#3207]
I've heard plenty of anecdotal complaints from folks who rely on video captions about sections of speech in different languages to the main soundtrack marked up as "[foreign speech]" or "[unintelligible]". 

Very often the language itself is a content detail (e.g. "[Hebrew Prayer]" is likely to communicate something about the characters or story rather more than "[foreign speech]").

Strictly speaking, these kinds of cases would appear to violate [WCAG SC 3.1.2](https://www.w3.org/TR/WCAG22/#language-of-parts), although that SC calls for a **machine readable** language code. 

I notice also that of the two common caption formats, SRT does **not** support marking up the language of individual cues, whereas VTT allows a lang attribute to be added to a "WebVTT cue language span" for this purpose. Setting this attribute would meet SC 3.1.2 without really resolving the problem for users who do not use screen readers.

In these cases, I dare say that the really important thing is to have the language mentioned **in the caption text itself**, so maybe it is better to consider this under "Info and Relationships".

Is it worth considering a small amendment to WCAG to explicitly handle this use case? 
Or does this fall into the category of "best practices"?

**Comment by alastc** [comment]
I think this falls into the area of 'quality of captions', which is above and beyond what WCAG currently requires.

I think it may be complex as well, in some cases identifying the language may be important. In some cases it might be a story choice to obscure that.

For example, in English captions the expectation might be that a certain language isn't understood or recognised. That might be a spoiler that people listening do not receive, and the captions give things away too early.

I've marked it as wcag.next, which is our label for reviewing things when we add guidelines to WCAG 3. I'll close the issue as we can't add it to WCAG 2.x, but it isn't lost.
